Fix repos get/update/delete for Git-CLI-enabled folders (#6122)
databricks/cli: .nextchanges/cli/repos-git-cli-folders.md

> Fixed `databricks repos get/update/delete` failing with `object at path "..." is not a repo` for Git-CLI-enabled folders, which the workspace API now reports as directories rather than repos.

Author: we should make note that the git cli enabled folders are a preview that can be enabled / disabled and alternatively the user can mitigate by turning off the feature

Author: Added a note that Git CLI is a preview and can be turned off in the workspace admin previews settings as an alternative mitigation. (b6988cc)

> Fixed `databricks repos get/update/delete` failing with `object at path "..." is not a repo` for Git-CLI-enabled folders, which the workspace API reports as directories rather than repos. Git CLI is a preview feature; as an alternative mitigation it can be turned off in the workspace admin previews settings.

Reviewer: Don't think we need this in the changelog
```suggestion
Fixed `databricks repos get/update/delete` failing with `object at path "..." is not a repo` for Git-CLI-enabled folders, which the workspace API reports as directories rather than repos.
```

Author: a given user who reads this may not know what git cli is is my main thing

Reviewer: Since this only affects/fixes issues for users who have Git-CLI-enabled folders, I think it's ok to drop the extra sentence. We don't need to communicate mitigations in the change log

Reviewer: ```suggestion
Fixed `databricks repos get/update/delete` failing with `object at path "..." is not a repo` for Git-CLI-enabled folders (currently in preview), which the workspace API reports as directories rather than repos.
```
